packages/next/build/webpack/config/blocks/css/index.ts

```diff
@@ -210,6 +214,24 @@ export const css = curry(async function css(
     )
   }
 
+  // Throw an error for Global CSS used inside of `node_modules`
+  fns.push(
+    loader({
+      oneOf: [
+        {
+          test: /\.css$/,
+          issuer: { include: [/node_modules/] },
```

Comment:
This would catch a `.css` file being imported from a `.css` file right 🤔 Might cause compat issues.

Reply:
Tests added in https://github.com/zeit/next.js/pull/9855. Seem to be passing!